### ISSUE TITLE
Ember server won't restart Compass for edits in files located outside of `app/styles` 

### ISSUE THREAD

**Issue by lolmaus** [quaertym/ember-cli-compass-compiler#41]
Hi!

I'm converting my Ember CLI project structure to pods. Thus, i'm moving my Sass partials outside of the `app/styles` folder.

When i first start `ember server`, all CSS is rendered correctly. Styles from partials outside the `app/styles` make their way into the browser.

Then, when i edit a Sass partial, Ember server detects the edit and restarts the build. The problem is that Compass is not executed during that build! :astonished:

In other words, Compass is only executed during builds initiated for Sass files located inside `app/styles`. This makes it impossible to put Sass code into pods.

Please advise.

**Comment by lolmaus** [comment]
I tried setting `sassDir: 'app'`, but it didn't help. :(

**Comment by lolmaus** [comment]
I think this limitation is introduced with this line (specifically, the `'css'`): https://github.com/quaertym/ember-cli-compass-compiler/blob/master/index.js#L8

I traced it to this Ember CLI file: https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js

...but was unable to grasp it. :( Need help from Ember CLI devs.

**Comment by lolmaus** [comment]
Found a relevant issue on Ember CLI: https://github.com/ember-cli/ember-cli/issues/2561

Closing this.

**Comment by quaertym** [comment]
:+1:
